cloudformation/tag-manager.yaml

```diff
@@ -284,6 +282,69 @@ Resources:
         Value: flexible
       - Key: App
         Value: tag-manager
+  TagManagerCloudfrontCachePolicy:
+    Type: "AWS::CloudFront::CachePolicy"
+    Properties:
+      CachePolicyConfig:
+        DefaultTTL: 86400
+        MaxTTL: 31536000
+        MinTTL: 0
+        Name: !Sub "tag-manager-cache-policy-${Stage}"
+        ParametersInCacheKeyAndForwardedToOrigin:
+          CookiesConfig:
+            CookieBehavior: "all"
+          EnableAcceptEncodingBrotli: false
+          EnableAcceptEncodingGzip: false
```

Comment:
Why not enable these?

Reply:
Let's make them true, I think I inherited these from another CFN and didn't have a reason to doubt them.

Reply:
Now set to true.